assets/minecraft/texts/splashes.txt

```diff
@@ -6,9 +6,95 @@ Made by Sake!
 Made by &flower!
 Made by Nexx!
 Made by Skye!
-Made by Laxla!
+Made by Laxystem!
+(We're not a laxlative :kittygun:)
```

Comment:
this would be confusing since it would have no context

Reply:
yes